src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+export interface ApiKeyModel {
+    organizationId?: string;
+    id?: string;
+    enabled?: boolean;
+    value?: string;
+    displayName?: string;
+    description?: string;
+    createdBy?: {};
+    createdDate?: number;
+    allowedIps?: string[];
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+    deniedIps?: string[];
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    privileges?: PrivilegeModel[];
+    resourceId?: string;
+}
+
+export enum Provider {
+    SALESFORCE = 'SALESFORCE',
+    SALESFORCE_SANDBOX = 'SALESFORCE_SANDBOX',
+    GOOGLE = 'GOOGLE',
+    OFFICE365 = 'OFFICE365',
+    SAML = 'SAML',
+    EMAIL = 'EMAIL',
+    OTHER = 'OTHER',
+}
+
+export interface IdAndDisplayNameModel {

```

Comment:
Same here, this interface is already defined in the GroupInterfaces

Reply:
Move it to BaseInterfaces.ts maybe?